.eslintrc.json

```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "airbnb-base"
+  "extends": "airbnb-base",
```

Comment:
Shouldn't we use https://github.com/dashevo/js-dpp/blob/v0.16-dev/.eslintrc ?

Reply:
I feel it is unnecessary (noplusplus for instance). And in all case we will have small diff. But do you wish me to add those dpp rules aswell ?

Reply:
Merged with our org wide set of rules :)